xblockutils/base_test.py

```diff
@@ -69,9 +69,9 @@ def wait_until_text_in(self, text, elem):
 
     def wait_until_exists(self, selector):
         wait = WebDriverWait(self.browser, self.timeout)
-        wait.until(lambda driver: driver.find_element_by_css_selector(selector), u"Selector '{}' should exist.")
+        wait.until(lambda driver: driver.find_element_by_css_selector(selector), u"Selector '{}' should exist.".format(selector))
 
-    def go_to_page(self, page_name, css_selector=None):
+    def go_to_page(self, page_name, css_selector=None, studio=False):
```

Comment:
@Kelketek It seems that the workbench supports arbitrary views. Do you think it would make sense to replace the `studio=False` parameter with `view_name='student_view'`?